packages/protocol/src/action-protocol/element-move.ts

```diff
@@ -0,0 +1,55 @@
+/********************************************************************************
+ * Copyright (c) 2024 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { Action } from './base-protocol';
+
+/**
+ * Action for triggering moving of elements. It is similar to `MoveAction` but is used for moving elements relative
+ * to their current position.
+ * The corresponding namespace declares the action kind as constant and offers helper functions for type guard checks
+ * and creating new `MoveElementRelativeAction`.
+ */
+export interface MoveElementRelativeAction extends Action {
```

Comment:
This action should not be part of the client-server protocol. Only a limited subset of client-side only actions are part of the protocol. The `MoveElementRealtiveAction` is defacto a feedback action. Dispatching this from the server-side does not make sense. (Movement changes on the server side should typically be operations)